core/services/pipeline/task.bridge.go

```diff
@@ -88,17 +89,23 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
 		requestData       MapParam
 		includeInputAtKey StringParam
 		cacheTTL          Uint64Param
+		reqHeaders        StringSliceParam
 	)
 	err = multierr.Combine(
 		errors.Wrap(ResolveParam(&name, From(NonemptyString(t.Name))), "name"),
 		errors.Wrap(ResolveParam(&requestData, From(VarExpr(t.RequestData, vars), JSONWithVarExprs(t.RequestData, vars, false), nil)), "requestData"),
 		errors.Wrap(ResolveParam(&includeInputAtKey, From(t.IncludeInputAtKey)), "includeInputAtKey"),
 		errors.Wrap(ResolveParam(&cacheTTL, From(ValidDurationInSeconds(t.CacheTTL), t.config.BridgeCacheTTL().Seconds())), "cacheTTL"),
+		errors.Wrap(ResolveParam(&reqHeaders, From(NonemptyString(t.Headers), "[]")), "reqHeaders"),
```

Comment:
Introducing a new feature requires to update documentation as well
(separate PR)
https://github.com/smartcontractkit/documentation

Reply:
Understood, made PR here https://github.com/smartcontractkit/documentation/pull/1122